RecoTracker/CkfPattern/python/GroupedCkfTrajectoryBuilderP5_cff.py

```diff
@@ -14,7 +14,7 @@
 Chi2MeasurementEstimatorForP5.nSigma = 4.
 Chi2MeasurementEstimatorForP5.MaxDisplacement = 100
 Chi2MeasurementEstimatorForP5.MaxSagitta=-1
-
+Chi2MeasurementEstimatorForP5.MinPtForHitRecoveryInGluedDet=100000
```

Comment:
an example of python spell
http://cmslxr.fnal.gov/dxr/CMSSW_8_0_5/search?q=MaxSagitta&case=true
gives you an idea of all the places where such an explicit spell can be cast